buildpack/build.go

```diff
@@ -289,6 +290,14 @@ func (b *Descriptor) readOutputFiles(bpLayersDir, bpPlanPath string, bpPlanIn Pl
 		}
 		br.MetRequires = names(bpPlanIn.filter(bpBuild.Unmet).Entries)
 
+		// set BOM files
+		if api.MustParse(b.API).AtLeast("0.7") {
```

Comment:
Is this version supposed to be `0.8`? I would have expected it to match here: https://github.com/buildpacks/lifecycle/pull/749/files#diff-5dce70063c4b1b07e611b3fcd32da0445a0b95970028c8992beb9c955e38320fR110

Reply:
This is the buildpack API - the sBOM work requires both a platform and buildpack api bump.